scraper.py

```diff
@@ -0,0 +1,40 @@
+﻿import glob
```

Comment:
just to make it more specific, might want to change the name of this file to `salesscraper.py` to differentiate it better from `cardscraper.py`.

Reply:
Also add a docstring at the top of the script.